media/js/firefox/firstrun/new-firstrun.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+    let scene = document.getElementById('scene');
+    let skipbutton = document.getElementById('skip-button');
+
+    let onVerificationComplete = () => {
+        scene.setAttribute('sign-in', 'true');
+        document.getElementById('sunrise').addEventListener('transitionend', function(event) {
+            if (event.propertyName === 'transform') {
+                window.setTimeout(function () {
+                    window.location.href ="https://github.com/";
+                    // Mozilla.UITour.showNewTab(); // TODO someday
```

Comment:
side note - it works now - the issue was permission to access uitour from `127.0.0.1:8111`